### PR TITLE
Update for FRENCH translation for Abous us extension

### PR DESCRIPTION
**Pull request by Galixte**
Update for FRENCH translation for common.php file

### CODE REVIEW

**Comment by Galixte** [comment]
Hi @Crizz0 ,

it's OK for me, could you take a look ?

File: language/fr/common.php [hunk 1]
```diff
@@ -20,26 +20,26 @@
...
+	'ACP_TERMS_OF_USE_ENABLE' 			=> 'Afficher les conditions d’utilisation',
+	'ACP_TERMS_OF_USE_ENABLE_EXPLAIN'	=> 'Affiche un lien vers les conditions d’utilisation sur la page « A propos ».',
 	'ACP_PRIVACY_ENABLE' 				=> 'Afficher la politique de la vie privée',
-	'ACP_PRIVACY_ENABLE_EXPLAIN'		=> 'Affiche un lien vers la politique de la vie privée sur la page "A propos".',
+	'ACP_PRIVACY_ENABLE_EXPLAIN'		=> 'Affiche un lien vers la politique de la vie privée sur la page « A propos ».',
 
-	'DESCRIPTION_PRIVACY'			=> 'Vous pouvez consulter la politique de la vie privée ici:',
-	'DESCRIPTION_TERMS_OF_USE'		=> 'Vous pouvez consulter les conditions d\'utilisation:',
-	'DESCRIPTION_PRIVACY_TERMS_OF_USE'=> 'Vous pouvez consulter les conditions d\'utilisation et la politique de la vie privée ici:',
+	'DESCRIPTION_PRIVACY'			=> 'Vous pouvez consulter la politique de la vie privée ici :',
```
Comment (Crizz0) [inline language/fr/common.php:40]: why did you add a space before : ?

**Comment by Galixte** [comment]
Hi @Crizz0,

i'm French, and in french language we have a space before and after a colon (:). Also, in phpBB template it's why the developpers use the {L_COLON} to take in consideration this rule with similar languages.

Best regards.
